src/pages/admin/training/list/page.tsx

```diff
@@ -17,8 +18,11 @@ export class ListTrainingPage extends React.Component<IProps, {}> {
   public render() {
     return (
       <div>
-        <TrainingTableComponent trainingList={this.props.trainingList}/>
-        <Link to={adminRouteEnums.training.edit}>Go to training Edit</Link>
+        <h1 className="text-center">Active Trainigs</h1>
+        <AutoSizer disableHeight={true}>
+          {({ width }) => <TrainingTableComponent width={width} trainingList={this.props.trainingList} />} 
+        </AutoSizer>
+        <Link to={adminRouteEnums.training.edit}>Go to training Edit</Link>  
```

Comment:
Right now when you click it go to a page with two options: Back to training, listBack to Dashboard.
I haven't took this line, this one is as always if you prefer I can change it.